### PR TITLE
Buildscript for remarkable v2.0.1

### PR DESCRIPTION
**Pull request by rajusah15**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@rajusah15 always mention the name of the package in the title of the PR, something like "Build script for <Package Name> for <Distro Name> <Distro Version>. You can also optionally add the package version if the script name has it (that is if the script is very specific to a version)

**Comment by rajusah15** [comment]
> 

Noted.

File: r/remarkable/remarkable_ubi_8.3.sh [hunk 1]
```diff
@@ -14,18 +16,21 @@
...
-export JAVA_HOME=/usr/lib/jvm/$(ls /usr/lib/jvm/ | grep -P '^(?=.*java-)(?=.*ppc64le)')
-echo "JAVA_HOME is $JAVA_HOME"
-# update the path env. variable 
-export PATH=$PATH:$JAVA_HOME/bin
+set -e
+PACKAGE_NAME=remarkable
+PACKAGE_VERSION=${1:-v2.0.1}
+PACKAGE_URL=https://github.com/jonschlinkert/remarkable.git
 
+yum install -y git npm
```
Comment (kandarpamalipeddi) [inline r/remarkable/remarkable_ubi_8.3.sh:24]: By default, yum installs node 10.
Please use the node 12 for build and validations.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

**Comment by kandarpamalipeddi** [comment]
Closing this PR, as it is renaming the no-existing file with remarkable script.

**Pull request closed without merging**